src/components/stepforms/widgets/VariableInputs/VariableChooser.vue

```diff
@@ -90,6 +98,13 @@ export default class VariableChooser extends Vue {
     }, []);
   }
 
+  /**
+  Return a readable value to display as tooltip
+  **/
+  readableValue(value: any) {
```

Comment:
```suggestion
  readableValue(value: any): string {
```
Et je sais pas si c'est une bonne chose de séparer string du reste. Je comprends que c'est plus lisible mais comment différencier `1` et `'1'`. Ca mériterait sinon un code couleur ou whatever non?